ui/src/app/version-info/components/version-info-panel.tsx

```diff
@@ -0,0 +1,103 @@
+import {DataLoader, SlidingPanel, Tooltip} from 'argo-ui';
+import * as React from 'react';
+import * as CopyToClipboard from 'react-copy-to-clipboard';
+import {VersionMessage} from '../../shared/models';
+
+import './version-info.scss';
+
+interface VersionPanelProps {
+    isShown: boolean;
+    onClose: () => void;
+    version: Promise<VersionMessage>;
+}
+
+export class VersionPanel extends React.Component<VersionPanelProps, {justCopied: boolean}> {
+    private readonly header = 'Argo CD Server Version';
+
+    constructor(props: VersionPanelProps) {
+        super(props);
+        this.state = {justCopied: false};
+    }
+
+    public render() {
+        return (
+            <DataLoader load={() => this.props.version}>
+                {version => {
+                    return (
+                        <SlidingPanel header={this.header} isShown={this.props.isShown} onClose={() => this.props.onClose()} hasCloseButton={true} isNarrow={true}>
+                            <div className='version-info-table argo-table-list'>
+                                {/* <div className='argo-table-list__head'>
+                                    <div className='row'>
+                                        <div className='columns small-4'>Tool</div>
+                                        <div className='columns small-4'>Version</div>
+                                    </div>
+                                </div> */}
+                                {this.buildVersionTable(version)}
+                            </div>
+                            <div className='version-copy-btn-container'>
+                                <Tooltip content='Copy all version info as JSON'>
+                                    <CopyToClipboard text={JSON.stringify(version, undefined, 4)} onCopy={() => this.onCopy()}>
```

Comment:
Could we leave out the `copy-to-clipboard` library? 

```{javascript}
onClick={() => {navigator.clipboard.writeText(TEXT_TO_COPY)}}
```

Most browsers support this now: https://caniuse.com/?search=clipboard
I don't think it's worth it to include a 37kB library for something this small.

Reply:
yes I will use that API. I recall when I was looking at clipboard options none of the ones I saw had great support so I guess I missed this one at 96%.